app/lib/proof_provider/keybase/config_serializer.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class ProofProvider::Keybase::ConfigSerializer < ActiveModel::Serializer
+  include RoutingHelper
+
+  attributes :version, :domain, :display_name, :username,
+             :brand_color, :logo, :description, :prefill_url,
+             :profile_url, :check_url, :check_path, :avatar_path,
+             :contact
+
+  def version
+    1
+  end
+
+  def domain
+    Rails.configuration.x.local_domain
+  end
+
+  def display_name
+    Setting.site_title
+  end
+
+  def logo
+    { svg_black: full_asset_url(asset_pack_path('media/images/logo_transparent_black.svg')), svg_full: full_asset_url(asset_pack_path('media/images/logo.svg')) }
+  end
+
+  def brand_color
+    '#282c37'
+  end
+
+  def description
+    Setting.site_short_description.presence || Setting.site_description.presence || I18n.t('about.about_mastodon_html')
+  end
+
+  def username
+    { min: 1, max: 30, re: Account::USERNAME_RE.inspect }
+  end
+
+  def prefill_url
+    params = {
+      provider: 'keybase',
+      token: '%{sig_hash}',
+      provider_username: '%{kb_username}',
+      user_agent: '%{kb_ua}',
+    }
```

Comment:
The validator wants there to be a username parameter (presumably, Mastodon should check to make sure this matches the logged-in user's name).

```json
{
  "status": {
    "code": 100,
    "desc": "missing or invalid inputs {\"prefill_url\":\"Error: Input https://mastodon.example.com/settings/identity_proofs/new?provider=keybase&provider_username=%{kb_username}&token=%{sig_hash}&user_agent=%{kb_ua} missing %{username}\"}",
    "name": "INPUT_ERROR"
  }
}
```

```suggestion
      username: '%{username}',
    }
```

Reply:
@xgess This wasn't in the original PR--has the validation on your side changed? We don't use the `username` param on that page on our side...

Reply:
I added the param anyway for the time being